config/locales/ru.yml

```diff
@@ -1096,7 +1097,8 @@ ru:
     reference: "Ссылка"
     refund: "Возврат"
     refund_amount_must_be_greater_than_zero:
-    refund_reasons: "Причины возврата"
+    refund_reasons: "Причины возврата денег"
+    return_reasons: "Причины возврата товара"
```

Comment:
Line 1280 is better, basically the meaning is same, but it's shorter, it doesn't deface header's navigation menu.